Resources/ModelingAndBinding-Starter/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+
+use_frameworks!
+
+pod 'IGListKit', :git => 'https://github.com/Instagram/IGListKit.git', :branch => 'master'
```

Comment:
Lol ya duh.

There's 2 versions of them: the starter and the finished. Not really an example, more like a working project for someone to go along w/ the guide.